common/shlibs

```diff
@@ -554,8 +554,8 @@ libicudata.so.67 icu-libs-67.1_1
 libicutu.so.67 icu-libs-67.1_1
 libicuuc.so.67 icu-libs-67.1_1
 libicutest.so.67 icu-libs-67.1_1
-libaspell.so.15 aspell-0.60_1
-libpspell.so.15 aspell-0.60_1
+libaspell.so.15 libaspell-0.60.8_2
+libpspell.so.15 libaspell-0.60.8_2
```

Comment:
I guess you could remove the duplicated line, I don't think there is a reason for that.

Reply:
libaspell vs libpspell

Reply:
oh sorry, looks good.